cg/store/filters/status_case_filters.py

```diff
@@ -274,6 +286,7 @@ class CaseFilter(Enum):
     BY_INTERNAL_ID_SEARCH: Callable = filter_cases_by_internal_id_search
     BY_NAME: Callable = filter_cases_by_name
     BY_NAME_SEARCH: Callable = filter_cases_by_name_search
+    BY_DATA_ANALYSES: Callable = filter_cases_by_data_analyses
```

Comment:
Should this be called `BY_WORKFLOWS`? When to use `workflow` which is the name of the StrEnum and when to use `data_analysis` which is the Sample attribute?

Reply:
`BY_WORKFLOWS` sounds most intuitive to me

Reply:
Changed!